amd64/src/semantic.rs

```diff
@@ -747,7 +755,7 @@ pub fn call(a: Rvalue) -> Result<(Vec<Statement>, JumpSpec)> {
         call ?, new_rip:64;
     });*/
     let stmts = rreil!{
-        call ?, (a);
```

Comment:
This is depends on the calling conventions. Functions can have multiple return values or return nothing. My plan is to replace the current call op with something that allows you to specify a whole vector of written and read R/Lvalues, but the code isn't ready for prime time yet.